config/systemd-monitor.json

```diff
@@ -0,0 +1,28 @@
+{
+	"plugin": "journald",
+	"pluginConfig": {
+		"source": "systemd"
+	},
+	"logPath": "/var/log/journal",
+	"lookback": "",
+	"bufferSize": 10,
+	"source": "systemd-monitor",
+	"conditions": [],
+	"rules": [
+		{
+			"type": "temporary",
+			"reason": "KubeletRestart",
+			"pattern": "Started Kubernetes kubelet."
+		},
+		{
+			"type": "temporary",
+			"reason": "DockerRestart",
+			"pattern": "Starting Docker Application Container Engine..."
```

Comment:
`s/Starting/Started`

Reply:
In our Debian OS machine. when docker daemon is restarted, the log printed is ` Started Docker Application Container Engine.`. Seems the trailing dot are redundant.

Reply:
> Is it possible the container runtime somehow gets stuck at the 1st log line and never reach the log line with Started? Detecting Starting can count for that.

This is reasonable. 

systemd will print logs with pattern `Started UNIT_FILE_DESCRIPTION`. Will confirm this later.

Reply:
@wangzhen127 According to the source code of systemd https://github.com/systemd/systemd/blob/master/src/core/service.c#L4288-L4301. Systemd will start two lines of log. When a service unit is started `Starting %s ...` and `Started %s .` when the job is done successfully. 

In case you mentioned above, i think `Starting %s ...` should be used. WDYT?

Reply:
I agree.

One caveat is that for kubelet, I don't see the `Starting...` log line on GKE VMs. So I used `Started...`. Not sure if it is related to how it is started by systemd (using `systemctl start`).

Reply:
> Not sure if it is related to how it is started by systemd (using systemctl start).

This should not be the truth. docker or containerd is also started with `systemctl start`. Can you double check this?

Reply:
I have double checked this.

Reply:
Well then, let's use `Started` for kubelet for now as actually this is just a hint for others end users. Users should adapt this pattern for their environment. We are in a heterogeneous world. :)